src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -1964,6 +1965,10 @@ namespace HVACUnitarySystem {
 					DataTotCapCurveIndex = GetDXCoilCapFTCurveIndex( UnitarySystem( UnitarySysNum ).CoolingCoilIndex, ErrFound );
 					DataIsDXCoil = true;
 				}
+				if ( UnitarySystem( UnitarySysNum ).CoolingCoilType_Num == Coil_CoolingAirToAirVariableSpeed ) {
+					DataTotCapCurveIndex = GetVSCoilCapFTCurveIndex( UnitarySystem( UnitarySysNum ).CoolingCoilIndex, ErrFound );
+					DataIsDXCoil = true;
+				}
```

Comment:
The VS DX coil never used the TotCapFTemp curve to adjust coil capacity during sizing. This should have been here all along.